### ISSUE TITLE
Incorrect parsing of dependencies

### ISSUE THREAD

**Issue by kgizdov** [aurapm/aura#436]
the package [lib32-qt4](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=lib32-qt4) has moved to AUR and it seems now it presents an issue for AURA - it cannot process its PKGBUILD correctly:

```bash
user@host $ sudo aura -Aux                                         
[sudo] password for user: 
aura >>= Fetching package information...
aura >>= Comparing package versions...
aura >>= AUR Packages to upgrade:
lib32-qt4 : 4.8.7-6 => 4.8.7-7

aura >>= Determining dependencies...
aura >>= Dependency checking failed for these reasons:
The dependency `lib32-fontconfiglib32-sqlitelib32-alsa-liblib32-glib2lib32-dbuslib32-openssl` could not be found. You may need to search for a package to satisfy it.
user@host $
```
I think the offending lines are:
```bash
depends=(lib32-{fontconfig,sqlite,alsa-lib,glib2,dbus,openssl}
         lib32-lib{png,tiff,mng,gl,sm,xrandr,xv,xi} $_pkgbasename)
```
where `a-{b,c}` is expanded to `a-ba-c` instead of to `a-b a-c`.

**Comment by fosskers** [comment]
This is being fixed.

**Comment by fosskers** [comment]
This has been fixed and will be included in `aura-1.3.8`.
